### DISCUSSION TITLE
For some reason Chrome did not write a file at `images/book/thumbnail/692675.png`.

### DISCUSSION THREAD

**Discussion by einwed** [spatie/browsershot#666] [Q&A]
Controller.php
`$path = 'images/book/thumbnail';
            $filename = $book->id . '.png';
            $file = $path . '/'. $filename;
            $view = view('book.book_thumbnail', ['book' => $book]);

            Browsershot::html($view->render())
                ->userAgent('Mozilla/5.0 (iPhone; CPU iPhone OS 11_0 like Mac OS X) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Mobile/15A372 Safari/604.1')
                ->windowSize(600, 315)
                ->save($file);`

ErrorMessage
`For some reason Chrome did not write a file at `images/book/thumbnail/692675.png`.
    Command
    =======
    {"url":"file:\/\/\/tmp\/53225323-0013538001661411531\/index.html","action":"screenshot","options":{"type":"png","path":"images\/book\/thumbnail\/692675.png","args":[],"viewport":{"width":600,"height":315},"displayHeaderFooter":false,"userAgent":"Mozilla\/5.0 (iPhone; CPU iPhone OS 11_0 like Mac OS X) AppleWebKit\/604.1.38 (KHTML, like Gecko) Version\/11.0 Mobile\/15A372 Safari\/604.1"}}
    Output
    ======`

yesterday, apt was updated. and I have this problem. when I put image by use 'Storage::put', It works. but when I use Browsershot, show this error message. how to fixed?

**Comment by neoteknic** [comment]
Same problem, browsershot stop working few month ago. I extracted the raw node command, but no error when I exec it, tring to find a debug mode...

**Comment by valkedev** [comment]
Same problem here:

```

Spatie\Browsershot\Exceptions\CouldNotTakeBrowsershot: For some reason Chrome did not write a file at /home/falke2/www/folder1/folder2/storage/app/cover_f0725c47-5667-40f0-9e86-9918b3b79a48.pdf. in /home/falke2/www/folder1/folder2/vendor/spatie/browsershot/src/Exceptions/CouldNotTakeBrowsershot.php:11

Stack trace:

#0 /home/falke2/www/bildy/caps-bi/vendor/spatie/browsershot/src/Browsershot.php(570): Spatie\Browsershot\Exceptions\CouldNotTakeBrowsershot::chromeOutputEmpty()
#1 /home/falke2/www/bildy/caps-bi/vendor/spatie/browsershot/src/Browsershot.php(497): Spatie\Browsershot\Browsershot->savePdf()
#2 /home/falke2/www/bildy/caps-bi/app/Exports/Pdf/CustomerSavingsReportExport.php(144): Spatie\Browsershot\Browsershot->save()

```

**Comment by code-crafter365** [comment]
Hi einwed, I am also getting the same issue. Did you solve this issue before?

**Reply by Merite15** [reply]
Same problem on windows machine

**Comment by oipablodias** [comment]
Just spent all day on this same issue. Fixed it with `npm install puppeteer@^10.4.0` wich is the one compatible with node v12.22.12 in my case.

**Reply by Merite15** [reply]
But i have node 16.15.0 and puppeteer 10.4.0 doesn't no longer supported
